cmake/developer_package/plugins/plugins.hpp.in

```diff
@@ -10,12 +10,19 @@
 #ifdef OPENVINO_STATIC_LIBRARY
 
 #include "cpp_interfaces/interface/ie_iplugin_internal.hpp"
+#include "openvino/core/extension.hpp"
+
+// The Macro used to create exyensions for static library
+#define OV_DEFINE_EXTENSION_CREATE_FUNCTION_DECLARATION(_OV_CREATE_EXTENSION_FUNC) \
+    INFERENCE_EXTENSION_API(void)                                                  \
```

Comment:
do we need export macro `INFERENCE_EXTENSION_API` to static library? I suppose - no